Code/GraphMol/MolDraw2D/DrawMol.cpp

```diff
@@ -684,6 +686,24 @@ void DrawMol::extractBrackets() {
             label, TextAlignType::MIDDLE, "connect",
             drawOptions_.annotationFontScale, topPt + (topPt - brkPt),
             DrawColour(0.0, 0.0, 0.0), textDrawer_);
+        if (brkPt.x < topPt.x) {
+          da->align_ = TextAlignType::START;
+        }
+        return da;
+      };
+
+      std::string label;
+      if (sg.getPropIfPresent("LABEL", label)) {
+        auto da = drawBottomLabel(label, *postShapes_[labelBrk], horizontal);
+        annotations_.emplace_back(da);
+      } else if (sg.getPropIfPresent("TYPE", label)) {
+        if (label == "GEN") {
+          // ChemDraw doesn't draw the GEN (type=generic) label.
+          continue;
+        }
+        // draw the lowercase type if there's no label to go there.
```

Comment:
why lowercase?
I'm not suggesting that it's wrong, just want to understand the decision

Reply:
The OP had it lowercase, as does ChemDraw, and the examples in the CTFile documentation do also. I didn’t see anywhere that explicitly said it should be lowercase, but I just inferred it from this evidence.

Reply:
ok